features/show/tabs.feature

```diff
@@ -23,5 +24,8 @@ Feature: Show - Tabs
       end
     """
     Then I should see two tabs "Overview" and "Details"
-    And I should see "tab 1"
-    And I should see "tab 2"
+    And I should see the element "#overview span"
+    And I should not see the element "#details span"
+    Then I follow "Details"
+    And I should not see the element "#overview span"
+    And I should see the element "#details span"
```

Comment:
@deivid-rodriguez , this was added by @gregbell  before tabs was added to AA, it is about navigation tabs, so this is naming gap,  I think it should be  separated to different step files